hera_cal/redcal.py

```diff
@@ -569,29 +569,23 @@ def compute_ubls(self, data, gain_sols):
             ubl_sols[blgrp[0]] = np.average(d_gp, axis=0)  # XXX add option for median here?
         return ubl_sols
 
-    def firstcal(self, data, df, wgts={}, sparse=False, mode='default', norm=True, medfilt=False, kernel=(1, 11)):
-        """Solves for a per-antenna delay by fitting a line to the phase difference between
-        nominally redundant measurements.  To turn these delays into gains, you need to do:
-        np.exp(2j * np.pi * delay * freqs)
-
-        Args:
-            data: visibility data in the dictionary format {(ant1,ant2,pol): np.array}
+    def _firstcal_iteration(self, data, df, f0=0.0, wgts={}, offsets_only=False,
```

Comment:
Why is f0=0.0 the default value?

Reply:
Because it's the default value in fft_dly (which was previously missing any f0 before I fixed it) and I feel like it's confusing to have different default values for passthrough kwargs.

I can't think of a better default value. The only real alternative is to pass in the whole of `freqs` and make this function pick out f0.